include/robot_localization/navsat_transform.h

```diff
@@ -287,6 +287,10 @@ class NavSatTransform
     //!
     ros::Duration transform_timeout_;
 
+    //! @brief When true, do not print warnings for tf lookup failures.
+    //!
+    bool tf_silent_failure_;
```

Comment:
Actually, sorry, small request: can we put this up with the other booleans? My original aim here was to group variables by data type and _then_ alphabetically, though I realise there are some variables that don't adhere to that standard here.

Reply:
Totally, done in 8c435d5.